src/test/java/com/puppycrawl/tools/checkstyle/DefaultLoggerTest.java

```diff
@@ -139,35 +138,26 @@ public void testNewCtorWithTwoParameters() {
     }
 
     @Test
-    public void testNullInfoStreamOptions() {
+    public void testNullInfoStreamOptions() throws Exception {
         final ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-        final IllegalArgumentException ex = assertThrows(IllegalArgumentException.class,
-                () -> new DefaultLogger(outputStream, (OutputStreamOptions) null),
-                "IllegalArgumentException expected");
-        assertWithMessage("Invalid error message")
-                .that(ex)
-                .hasMessageThat()
-                        .isEqualTo("Parameter infoStreamOptions can not be null");
+        final DefaultLogger logger = new DefaultLogger(outputStream, OutputStreamOptions.NONE);
```

Comment:
Please keep nulls for ctors, and expect null to be working as none

Reply:
Done